mmdet3d/models/utils/gaussian.py

```diff
@@ -86,24 +93,28 @@ def gaussian_radius(det_size, min_overlap=0.5):
     return min(r1, r2, r3)
 
 
-def get_ellip_gaussian_2D(heatmap, center, radius_x, radius_y, k=1):
+def get_ellip_gaussian_2D(heatmap: Tensor,
+                          center: List[int],
+                          radius_x: int,
+                          radius_y: int,
+                          k: int = 1) -> Tensor:
     """Generate 2D ellipse gaussian heatmap.
 
     Args:
         heatmap (Tensor): Input heatmap, the gaussian kernel will cover on
             it and maintain the max value.
-        center (list[int]): Coord of gaussian kernel's center.
+        center (List[int]): Coord of gaussian kernel's center.
         radius_x (int): X-axis radius of gaussian kernel.
         radius_y (int): Y-axis radius of gaussian kernel.
-        k (int, optional): Coefficient of gaussian kernel. Default: 1.
+        k (int): Coefficient of gaussian kernel. Default: 1.
```

Comment:
```suggestion
        k (int, optional): Coefficient of gaussian kernel. Default: 1.
```
These arguments which have default values are optional. Please check all the files.

Reply:
It seems that only default value is `None` should be optional in the latest version.